lib/iam_ssoe_oauth/session_manager.rb

```diff
@@ -62,6 +63,13 @@ def create_user_session
       raise e
     end
 
+    def add_missing_fediamassur_level(iam_profile)
+      if iam_profile[:fediamassur_level].blank?
+        iam_profile[:fediamassur_level] = iam_profile[:fediam_aal] == '2' && iam_profile[:fediam_ial] == '2' ? 3 : 0
```

Comment:
I had to read it a couple of times before I realized the first part was a tenary argument:
```suggestion
        iam_profile[:fediamassur_level] = (iam_profile[:fediam_aal] == '2' && iam_profile[:fediam_ial] == '2') ? 3 : 0
```